lib/tasks/update_related_links_from_json.rake

```diff
@@ -0,0 +1,48 @@
+namespace :content do
+  desc "Updates suggested related links for content from a JSON file"
+  task :update_related_links_from_json, [:json_path] => :environment do |_, args|
+    puts 'Reading and parsing data...'
+
+    url = URI.parse(args[:json_path]) rescue false
+    file =
+      if url.is_a?(URI::HTTP) || url.is_a?(URI::HTTPS)
+        url.open(&:read)
+      else
+        File.read(args[:json_path])
+      end
+
+    json = JSON.parse(file)
+
+    @failed_content_ids = []
```

Comment:
If something fails, this could be filled with 500k content ids. Do you think this is an issue?

Reply:
This could be an issue, especially if it were automated. It will be run manually to begin with, and it would be useful to record all of the failed content ids in a central place so that only those content items can be retried later on.